source/js/ellipsis-util.js

```diff
@@ -0,0 +1,99 @@
+var ellipsis = require("html-ellipsis");
+var characterPerLine = require("./characters-per-line");
+
+var EllipsisUtil = {
+  /***
+   * @param {Object} lineStyles: Node with DOM elements that has a string
+   * @return {Boolean}
+   */
+  hasSpecificHeight: function(lineStyles) {
+    return lineStyles.height != "auto" && parseInt(lineStyles.height) != 0;
+  },
+
+  /***  
+   * @param {Node} element Node element.
+   * @return {Integer} Returns a font size.
+  * */
+  getFontSize: function(element) {
+    return parseInt(window.getComputedStyle(element).getPropertyValue("font-size"));
+  },
+
+  /***
+   * @summary Returns the width of the object 
+   * @param {Node} textLines: Node with DOM elements that has a string
+   * @return {Integer} width
+   */
+  getWidth: function(textLine) {
+    if(textLine.innerText === "") {
+      return 0;
+    }
+  
+    var lineStyles = window.getComputedStyle(textLine);
+
+    if(EllipsisUtil.hasSpecificHeight(lineStyles)) {
+      return parseInt(lineStyles.width);
+    }
+  },
+
+  /***
+   * @summary Returns the width of the object's parent 
+   * @param {Node} textLines: Node with DOM elements that has a string
+   * @return {Integer} parent's width
+   */
+  getParentsWidth: function(textLine) {
```

Comment:
parameter "textLine" is not describe in the docBlock

Reply:
done